src/components/RoomPortal.js

```diff
@@ -0,0 +1,40 @@
+import { Button, Input } from "antd";
+
+export default function RoomPortal() {
+  return (
+    <div className="portal-wrapper">
+      <div className="box-back">
+        <h3 className="rooms-title">Rooms</h3>
+      </div>
+
+      <div className="box-front">
+        <Button type="primary" className="create-btn">
+          Create
+        </Button>
+        <p className="or-text">OR</p>
+
+        <form name="basic" className="id-form">
```

Comment:
We should make sure and check the accessibility of an Ant Design component, test with a screen reader, etc.